lib/components/narrative/narrative-itineraries.js

```diff
@@ -233,17 +236,21 @@ class NarrativeItineraries extends Component {
       // If no duplicate, push full itinerary to output
       if (duplicateIndex === -1) {
         updatedItineraries.push(updatedItinerary)
-      } else {
+        // Only process itineraries less than 24 hours in the future
+      } else if (
+        updatedItinerary.startTime <
+        // 24 hours in milliseconds
+        Date.now() + 24 * 60 * 60 * 1000
```

Comment:
This won't work well if we're using the website in a time zone other than the time zone of the agency. Do we have easy access to the agency's timezone somehow that we could use with Datefns to do this properly?

Reply:
Good call! Luckily the leg provides a `agencyTimeZoneOffset` which we can combine with `new Date().getTimezoneOffset()` to ensure we're doing all time math in UTC!